### PR TITLE
Prepare the README for moving the kernel out.

### PR DESCRIPTION
**Pull request by ticki**
Related to: #731

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
-<img alt="Redox" height="90" src="https://github.com/redox-os/assets/raw/master/logo.png">
+<p align="center">
+<img alt="Redox" height="120" src="https://github.com/redox-os/assets/raw/master/logo.png">
+</p>
 
-**Redox** is an operating system written in pure Rust, designed to be secure and free. The website can be found at https://www.redox-os.org.
+**Redox** is an operating system written in Rust language, a language with focus on safety and high performance. Redox, following the microkernel design, aims to be secure, usable, and free. Redox is inspired by previous kernels and operating systems, such as SeL4, Minix, and BSD.
```
Comment (jackpot51) [inline README.md]: It is redundant to say in Rust language, a language - it could be in Rust, a language

Comment (jackpot51) [inline README.md]: Perhaps add Plan 9 to the list?

File: README.md [hunk 2]
```diff
@@ -29,6 +33,44 @@ Please make sure you use the **latest nightly** of `rustc` before building (for
 <img alt="Redox" height="150" src="https://github.com/redox-os/assets/raw/master/screenshots/Boot.png">
 <img alt="Redox" height="150" src="https://github.com/redox-os/assets/raw/master/screenshots/start.png">
 
+## <a name="ecosystem"> Ecosystem </a>
```
Comment (jackpot51) [inline README.md]: Amazing, thanks for this

File: README.md (hunk 1 above)
Comment (ticki) [inline README.md]: yeah

Comment (ticki) [inline README.md]: done

File: README.md [hunk 3]
```diff
@@ -29,6 +33,44 @@ Please make sure you use the **latest nightly** of `rustc` before building (for
...
+| [libpager](/redox-os/libpager)                            | [**@ticki**](/ticki)
+| [magnet (future package manager)](/redox-os/magnet)       | [**@ticki**](/ticki)
+| [netutils](/redox-os/netutils)                            | [**@jackpot51**](/jackpot51)
+| [orbclient](/redox-os/orbclient)                          | [**@jackpot51**](/jackpot51)
+| [orbdata](/redox-os/orbdata)                              | [**@jackpot51**](/jackpot51)
+| [orbital](/redox-os/orbital)                              | [**@jackpot51**](/jackpot51)
+| [orbtk](/redox-os/orbtk)                                  | [**@stratact**](/stratact)
+| [orbutils](/redox-os/orbutils)                            | [**@jackpot51**](/jackpot51)
+| [pkgutils (current package manager)](/redox-os/pkgutils)  | [**@jackpot51**](/jackpot51)
+| [playbot (internal REPL bot)](/redox-os/platbot)          | [**@ticki**](/ticki)
```
Comment (bjorn3) [inline README.md]: s/platbot/playbot
also it opens as https://github.com/redox-os/redox/blob/master/redox-os/platbot in my browser

**Pull request merged**